lib/common/core/CmdParser.cpp

```diff
@@ -591,10 +597,7 @@ int CmdLineArgsParser::Parse(__in LPWSTR oneArg) throw()
                 PrintUsage();
                 return -1;
             }
-            else
-            {
-                ParseFlag();
-            }
+            ParseFlag();
```

Comment:
I'm a fan of not having redundant else blocks in cases like this.  When an else block like this exists and code exists after it it can look like the else block doesn't necessarily always execute with the remaining code, and requires a bit more reading for full comprehension of the code.  Being terse and concise is good for readability.